### ISSUE TITLE
bug:

### ISSUE THREAD

**Issue by syaqil520** [LazyVim/LazyVim#4775]
### Did you check docs and existing issues?

- [x] I have read all the LazyVim docs
- [x] I have updated the plugin to the latest version before submitting this issue
- [x] I have searched the existing issues of LazyVim
- [x] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

v0.10.2

### Operating system/version

MacOS 15.0.1 and Arch linux 

### Describe the bug

After updating to new lazyvim, keep having error after installing new plugin and reopen the neovim.
 ![Image](https://github.com/user-attachments/assets/80d4666d-6fda-46e8-89ca-d63d9bd7d71d)
![Image](https://github.com/user-attachments/assets/31ed3587-597a-4a6e-8534-3ad97fae39dd)

the error will then keep popping until i close and reopen it again. 

seem like coming from the new opt.statuscolumn setting for the snacks.nvim as after i set the option to opt.statuscolumn="", the issue never happen

### Steps To Reproduce

1. installl new plugin 
2. close and open nvim 
3. after plugin finish install error will keep popping up 
4. close nvim and reopen 
5. nvim open like usual 

### Expected Behavior

1. install new plugin 
2. close and open nvim 
3. plugins is installed and able to be used like usual 

### Repro

```lua
vim.env.LAZY_STDPATH = ".repro"
load(vim.fn.system("curl -s https://raw.githubusercontent.com/folke/lazy.nvim/main/bootstrap.lua"))()

require("lazy.minit").repro({
  spec = {
    { "LazyVim/LazyVim", import = "lazyvim.plugins" },
    -- add any other plugins here
    { import = "lazyvim.plugins.extras.dap.core" },

  },
})
```

**Comment by dpetka2001** [comment]
So, I was able to reproduce it, but could not understand what was causing this. After quite a lot of trial and error, I tried changing in `options.lua` the setting to `opt.statuscolumn = [[%!v:lua.Snacks.statuscolumn()]]` and it seems to be working. I'm not making a PR and waiting for @folke to see this because I don't quite understand why this happens. The global `Snacks.whatever` is supposed to return the required module under the hood.
